python/comparison.cpp

```diff
@@ -0,0 +1,82 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// Copyright (c) 2020 Scipp contributors (https://github.com/scipp)
+/// @file
+/// @author Simon Heybrock
+#include "detail.h"
+#include "docstring.h"
+#include "pybind11.h"
+
+#include "scipp/dataset/dataset.h"
+#include "scipp/variable/comparison.h"
+
+using namespace scipp;
+using namespace scipp::variable;
+using namespace scipp::dataset;
+
+namespace py = pybind11;
+
+template <class T> Docstring docstring_comparison(const std::string op) {
+  return Docstring()
+      .description("Comparison returning the truth value of " + op +
+                   "element-wise.")
+      .raises("If the units of inputs are not the same, or if the dtypes of "
+              "inputs are not double precision floats.")
+      .returns("Booleans that are true if " + op + ".")
+      .rtype<T>()
+      .template param<T>("x", "Input left operand.")
+      .template param<T>("y", "Input right operand.");
```

Comment:
@OwenArnold @DanNixon What is peoples oppinion on such a Sphinx wrapper? Is it useful, or should we stick with the native syntax?

Reply:
:+1: It is nicer than writing the format by hand and it helps with consistent formatting.